src/controllers/periods.ts

```diff
@@ -1,18 +1,63 @@
-import { Request, Response } from 'express';
 import PeriodModel from '@entities/Period';
-import { QueryInput } from '@shared/inputs';
 import { getQuerySort } from '@shared/functions';
+import { QueryInput } from '@shared/inputs';
+import { Request, Response } from 'express';
+import { StatusCodes } from 'http-status-codes';
 
-// get periods
-const getPeriods = async (
+export const all = async (
   req: Request<any, QueryInput, any>,
   res: Response
 ): Promise<Response> => {
   const response = await PeriodModel.paginate({
     ...req.query,
     sort: getQuerySort(req.query),
   });
-  return res.status(200).json(response);
+  return res.status(StatusCodes.OK).json(response);
 };
 
-export default { getPeriods };
+export const single = async (
+  req: Request<any, QueryInput, any>,
+  res: Response
+): Promise<Response> => {
+  let period = await PeriodModel.findById(req.params.periodId);
+  if (!period) return res.status(StatusCodes.INTERNAL_SERVER_ERROR);
```

Comment:
it is a small thing but why are we using `let` instead of `const` here? and maybe we should return `NOT_FOUND` in cases like this?

Reply:
No reason. Will fix.